config/config.md

```diff
@@ -117,19 +117,10 @@ The following config option are provided by the OpenHIM. All of these options ha
     // To enable daily and weekly reporting
     "enableReports": true
   },
-  "visualizer": {
-    // To enable the visualizer API
-    "enableVisualizer": true,
-    // The minimum duration to use for the event. If the event's duration is
-    // less than this specified period, then the end time will be adjusted
-    // in order to match
-    "minimumEventPeriodMillis": 100,
-    // Should orchestration and route timestamps be normalized to the
-    // transaction time. This may be useful if you have mediators running on
-    // other servers
-    "enableTSNormalization": true,
-    // A buffer period for orchestration timestamps
-    "orchestrationTSBufferMillis": 100
+  "events": {
+    // Should event timestamps be normalized to the transaction time.
+    // This will be useful if you have mediators running on other servers
+    "enableTSNormalization": true
```

Comment:
If a mediator is running on another server, the reported timestamps don't always line up with core - so what this does is normalize those timestamps relative to core - ie. if a mediator orchestration takes 200ms, the events will be set relative to core's start time (orchestration event start == core start and orchestration event end == core start + 200ms), rather than the "real" orchestration time.

In an ideal world all servers would be running on the same time, but unfort that's just not always the case, even with ntp enabled, they might drift slightly before syncing up again. I think this was the case when we added this in (i believe a couple mins before a demo!!).

I think even a few ms out made for ugly results on the visualizer.